doc/src/Modify_contribute.rst

```diff
@@ -209,13 +209,13 @@ packages in the src directory for examples. If you are uncertain, please ask.
   A LaTeX citation is stored in a variable at the top of the file and a
   single line of code that references the variable is added to the
   constructor of the class.  Whenever a user invokes your feature from
-  their input script, this will cause LAMMPS to output the citation to a
-  log.cite file and prompt the user to examine the file.  Note that you
-  should only use this for a paper you or your group authored.
-  E.g. adding a cite in the code for a paper by Nose and Hoover if you
-  write a fix that implements their integrator is not the intended
-  usage.  That kind of citation should just be in the doc page you
-  provide.
+  their input script, this will cause LAMMPS to output a reminder to
+  cite that reference.  Note that you should only use this for the most
```

Comment:
Should we say it is allowable to make the summary a single paper (one line) but the full citation can be multiple (only if necessary) ?

Reply:
I don't think we need to explicitly state that, but it would be possible to allow this. I don't really see this as an issue for publications, but it could be for a feature that includes external code with its own citation, website, or DOI for the software.
In that case and single line followed by two bibtex entries would certainly be a good solution.